pkgs/development/libraries/rapidfuzz-cpp/default.nix

```
@@ -33,6 +34,13 @@ stdenv.mkDerivation (finalAttrs: {
     catch2_3
   ];
 
+  passthru = {
+    tests = {
+      /** `python3Packages.levenshtein` crucially depends on `rapidfuzz-cpp` */
+      levenshtein = python3Packages.levenshtein;
```

```suggestion
      levenshtein = python3Packages.levenshtein.override {
        rapidfuzz-cpp = finalAttrs.finalPackage;
      };
```
I would prefer to leave the `passthru.tests` attrset expanded (un-collapsed) such that it would be easier to include more tests in the future.